src/renderer/components/reader/ReaderFooter.tsx

```diff
@@ -7,19 +7,25 @@
 
 import * as React from "react";
 
-import * as styles from "readium-desktop/renderer/assets/styles/reader-app.css";
-
 import { Translator } from "readium-desktop/common/services/translator";
 import { lazyInject } from "readium-desktop/renderer/di";
 
 import * as ArrowRightIcon from "readium-desktop/renderer/assets/icons/baseline-arrow_forward_ios-24px.svg";
 import * as ArrowLeftIcon from "readium-desktop/renderer/assets/icons/baseline-arrow_left_ios-24px.svg";
 
+import { Publication } from "r2-shared-js/dist/es6-es2015/src/models/publication";
+
 import SVG from "readium-desktop/renderer/components/utils/SVG";
 
+import * as classnames from "classnames";
+import * as styles from "readium-desktop/renderer/assets/styles/reader-app.css";
+
 interface Props {
     navLeftOrRight: (left: boolean) => void;
     fullscreen: boolean;
+    currentLocation: any;
```

Comment:
Instead of the `any` type, could this be `currentLocation: LocatorExtended | undefined;`?
(with `import { LocatorExtended } from "@r2-navigator-js/electron/renderer/index";`)